packages/cfd/src/Containers/mt5-migration-modal/mt5-migration-back-side-content.tsx

```diff
@@ -1,113 +1,138 @@
-import React from 'react';
-import { Button, Checkbox, Modal, Text, StaticUrl, Icon } from '@deriv/components';
+import { Button, Modal, Text, PasswordInput, FormSubmitButton, PasswordMeter, Loading } from '@deriv/components';
 import { useMT5SVGEligibleToMigrate } from '@deriv/hooks';
-import { CFD_PLATFORMS, DBVI_COMPANY_NAMES } from '@deriv/shared';
+import { CFD_PLATFORMS, WS, validLength, validPassword, getErrorMessages } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
-import { Localize } from '@deriv/translations';
-import getMigrationModalDetails from '../../Constants/mt5-migration-modal-content';
+import { Localize, localize } from '@deriv/translations';
+import React from 'react';
 import { useCfdStore } from '../../Stores/Modules/CFD/Helpers/useCfdStores';
 import { useMT5MigrationModalContext } from './mt5-migration-modal-context';
+import { Formik, FormikErrors, FormikHelpers, FormikProps } from 'formik';
+import { TCFDPasswordFormValues } from 'Containers/cfd-password-modal';
 
 const MT5MigrationBackSideContent = observer(() => {
-    const { ui, common } = useStore();
+    const { ui, common, client } = useStore();
     const { toggleMT5MigrationModal, setMT5MigrationModalEnabled, is_mobile } = ui;
+    const { email } = client;
     const { setAppstorePlatform } = common;
-    const { enableCFDPasswordModal, mt5_migration_error, setJurisdictionSelectedShortcode } = useCfdStore();
+    const { setJurisdictionSelectedShortcode, setSentEmailModalStatus, submitMt5Password } = useCfdStore();
+    const { getEligibleAccountToMigrate } = useMT5SVGEligibleToMigrate();
+    const { setShowModalFrontSide, setShowMigrationError } = useMT5MigrationModalContext();
+    const formik_ref = React.useRef<FormikProps<TCFDPasswordFormValues>>(null);
 
-    const { getEligibleAccountToMigrate, eligible_account_to_migrate_label } = useMT5SVGEligibleToMigrate();
-    const { setShowModalFrontSide } = useMT5MigrationModalContext();
-    const [is_checked, setIsChecked] = React.useState(false);
+    const initial_values = {
```

Comment:
should we add types here for initial_values? or we can use as const?

Reply:
sure.. will add it.. 😄